.github/workflows/check.yaml

```diff
@@ -15,22 +15,82 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  lint-unit:
-    uses: canonical/bootstack-actions/.github/workflows/lint-unit.yaml@v2
+  lint:
+    name: Lint
     strategy:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.10"]
-    with:
-      python-version: ${{ matrix.python-version }}
-      tox-version: "<4"
+    runs-on: ubuntu-latest
```

Comment:
maybe this should be 22.04 to follow the other repos?

Reply:
thanks, updated :)